app/experimenter/experiments/tests/test_views.py

```diff
@@ -220,6 +220,136 @@ def test_list_view_lists_experiments_with_default_order_no_archived(self):
+                        "start_date_range_after": start_range_date,
+                        "start_date_range_before": end_range_date,
+                    }
+                ),
+            ),
+            **{settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+
+        context_1 = response_1.context[0]
+
+        self.assertEqual(set(context_1["experiments"]), set([exp_1]))
+
+        response_2 = self.client.get(
+            "{url}?{params}".format(
+                url=reverse("home"),
+                params=urlencode(
+                    {
+                        "show_experiment_changes": "pauses_between",
+                        "pause_date_range_after": start_range_date,
+                        "pause_date_range_before": end_range_date,
+                    }
+                ),
+            ),
+            **{settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+
+        context_2 = response_2.context[0]
+
+        self.assertEqual(set(context_2["experiments"]), set([exp_1, exp_2]))
+
+        response_3 = self.client.get(
+            "{url}?{params}".format(
+                url=reverse("home"),
+                params=urlencode(
+                    {
+                        "show_experiment_changes": "ends_between",
+                        "end_date_range_after": start_range_date,
+                        "end_date_range_before": end_range_date,
+                    }
+                ),
+            ),
+            **{settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+
+        context_3 = response_3.context[0]
+
+        self.assertEqual(set(context_3["experiments"]), set([exp_2, exp_4]))
+
+        response_4 = self.client.get(
+            "{url}?{params}".format(
+                url=reverse("home"),
+                params=urlencode(
+                    {
+                        "show_experiment_changes": "starts_between",
+                        "start_date_range_after": start_range_date,
+                        "start_date_range_before": "",
+                    }
+                ),
+            ),
+            **{settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+
+        context_4 = response_4.context[0]
+
+        self.assertEqual(set(context_4["experiments"]), set([exp_1, exp_3]))
+
+        response_5 = self.client.get(
+            "{url}?{params}".format(
+                url=reverse("home"),
+                params=urlencode(
+                    {
+                        "show_experiment_changes": "starts_between",
+                        "start_date_range_after": "",
+                        "start_date_range_before": end_range_date,
+                    }
+                ),
+            ),
+            **{settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+
+        context_5 = response_5.context[0]
+
+        self.assertEqual(
+            set(context_5["experiments"]), set([exp_1, exp_2, exp_4])
+        )
```

Comment:
Nit: this might be better as a separate `TestCase`, with all the experiment creation done in the `setUp`/`setUpClass` and each request a separate method. But I see you're following the pattern of other tests so it's not a big deal.

Reply:
@glasserc did you have any thoughts on the new tests written? @jaredkerim preferred me not to make a separate class but instead make a custom `set_up_date_tests` method to call in each different method.

Reply:
That's fine with me.